src/index.html

```diff
@@ -11,7 +11,137 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="head">
+        <legend>
+          Personal information
+        </legend> 
+        <div>
+          <label class="form">
```

Comment:
rename to field, sounds more logical

```suggestion
          <label class="field">
```